### DISCUSSION TITLE
Error: Invalid command name "customlib::SetBaseRoot"

### DISCUSSION THREAD

**Discussion by MiguelSC98** [Anura3D/Anura3D_OpenSource#216] [Q&A]
I have recently started using Anura3D and followed the software installation and tutorial diligently. However, I encountered an issue that I would like to bring to your attention.

I am using GID version 16.0.7 in conjunction with the latest release of Anura3D_2023. When I compiled Anura 3d. I obtained 0 erros and 917 warnings. 
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/4fecf207-c705-4432-9cbf-dc152871af9b)
But, at the end. I could install Anura3d in GID
Following the tutorial examples, I attempted to create a model and generate Anura3D files. Unfortunately, each time I tried, I received the following error message:
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/f4a212d7-b524-4e11-867d-aba919e33baf)
I would  appreciate your help in identifying the root cause of this issue and let me know if these 917 warnings do not affect in my results.
Thanks

**Comment by Abbki** [comment]
@MiguelSC98 @acrodrigud 
Check out the following discussion:
https://github.com/Anura3D/Anura3D_OpenSource/discussions/134#discussioncomment-7422480
